QUICKSTART.md

```diff
@@ -48,6 +47,10 @@ git remote add heroku git@heroku.com:appinfive.git
 vim Gemfile
   add
     gem 'shopify_app'
+    
+or simply run the following command 
+
+$ echo "gem 'shopify_app', '~> 7.0.0'" >> Gemfile
```

Comment:
Lets just replace the existing instruction with this (remove the vim stuff). Can you remove the version number from the echo?

I added a note on master about using the latest version so we're covered there now.